src/node/mod.rs

```diff
@@ -44,7 +44,7 @@ pub use waveshaper::*;
 mod stereo_panner;
 pub use stereo_panner::*;
 
-pub(crate) const TABLE_LENGTH_USIZE: usize = 2048;
+pub(crate) const TABLE_LENGTH_USIZE: usize = 8192;
 pub(crate) const TABLE_LENGTH_BY_4_USIZE: usize = TABLE_LENGTH_USIZE / 4;
 // 2048 casts without loss of precision cause its mantissa is 0b0
 #[allow(clippy::cast_precision_loss)]
```

Comment:
You should check this comment, does it uphold for the new value?

Reply:
good shot, no idea actually... will try to understand what that means exactly
(...I would be quite surprised that we can loose precision with such small numbers but...)

Reply:
Actually, I'm really not a computer scientist, but I don't understand that comment, it seems wrong to me

1. If we agree a floating point number is represented as: `±mantissa * 2^exponent`, if the mantissa is 0 (...or `0b0`), then the represented number is 0 (as far as I know `0 * x` will always be 0, whatever the exponent)
2. The mantissa seems to be generally represented by 23 bits for f32, so _any_ integer up (or down) to `2^23` can be perfectly represented with an exponent of 0 (without even considering that the number we are talking about is a power of two)

Or am I missing something?

Reply:
Actually these directives where not even required by clippy (at least in its CI config) so I just removed them